src/App.tsx

```diff
@@ -49,10 +44,42 @@ const App: React.FC = () => {
     const owner = pathArray[3];
     const repo = pathArray[4];
     setRepoName(repo);
+    // await getOldTree(owner, repo);
+    // if (oldTree) {
     await makeRequest(owner, repo);
+    // }
   };
 
   const makeRequest = async (owner: String, repo: String) => {
+    let oldTree: oldTree[] | null = null;
+
+    // Previous Comments
```

Comment:
remove useless comment

Reply:
@ralph-dev all segments in makeRequest has a comment above them... should I remove all of them or just Previous Comment?

Reply:
Previous comment provides no value, thus it should be removed. This rule applies to any comment that provides no value